### DISCUSSION TITLE
由于丁香园批评国内的防疫政策和中医，现在被全网禁言。

### DISCUSSION THREAD

**Discussion by ghost** [WeNeedHome/SummaryOfLoanSuspension#1114] [General]
由于丁香园批评国内的防疫政策和中医，现在被全网禁言。这是国内最好的医学科普平台，对循证医学的普及，功莫大焉，代表的是中国医生的学术和道德水平，看来新时代中国容不得半点不同声音。防疫政策是要坚定不移的贯彻下去了，想要疫情走那是不可能的，不然让贪官们怎么活？🤣

再过几个月世卫组织即将宣布新冠疫情将从世界上消失，而伟大的中国人民还在坚定不移的抗击新冠肺炎疫情😁，中国人民最后取得的抗议成果如何，那就让我们拭目以die吧。

![image](https://user-images.githubusercontent.com/110806464/183795986-764da40f-28bd-4c2e-97b1-e15b287af6a5.png)
![image](https://user-images.githubusercontent.com/110806464/183796021-ca5e6bdd-5e89-4165-8624-5796b0d77e08.png)
![image](https://user-images.githubusercontent.com/110806464/183796037-87b21be0-e92a-4387-83a4-91eaf27255f9.png)
![image](https://user-images.githubusercontent.com/110806464/183796044-5447736e-936d-4da9-8997-9e2ea35d4b35.png)
![image](https://user-images.githubusercontent.com/110806464/183796061-fa417e62-5e5e-4b76-b045-c2a067859188.png)

**Comment by super-aviator** [comment]
头疼捂嘴，脚疼捂嘴

**Reply by Hughlaud** [reply]
政治账：1>1400 000 000

**Reply by xunfeng2zkj** [reply]
丁香园屁股还是有点歪的

**Comment by LeeNNN** [comment]
对对对，丁香医生们拖完地还会用嘴舔一遍，擦完马桶能直接喝马桶水

**Comment by macleer** [comment]
虽然有点牵强，但丁香园确实不干净

**Comment by Shenkeis** [comment]
为啥不是因为丁香园挑拨女权，宣传男性结扎无危害呢

**Reply by CKwuliuqi** [reply]
丁香医生真不是啥好东西，我在知乎评论了几句丁香医生做专业知识平台跟散播意识形态并不矛盾直接给我整封号了，资本主义的力量也不可小觑啊

**Comment by MYES0622** [comment]
只能说丁香医生好似

**Comment by ColoCoffee** [comment]


**Reply by ghost** [reply]
老子乱刀劈死你全家，你全家十八代祖坟被人掘了吗？你以为你是在微博是吧？你跟你妈的个篮子样。你爹我不惯着你，你这种人出来老子第一个乱刀砍死的就是你，蠢的跟你妈的比一样，还有逼脸在这叫，你全家女性是母狗被人轮奸了。老子砍你全家的头，啥寄吧玩意。

**Reply by ColoCoffee** [reply]
你这种死妈的东西就别来污染github环境了，跟你妈疯狗一样上来就咬，你祖上是妓女出生吧？让你的嘴和你妈的b一样脏？你是不是你妈被轮奸的时候生下来的让你变得这么低能？脑袋在你妈被轮奸的时候被撞坏了？来砍老子？你有这本事么？你爹马上去东京，你这fw估计连本护照都办不下来吧？

**Reply by ghost** [reply]
> 我要笑死了，世卫宣布新冠消失？你是多低能

你全家女性都被老子爆杀，老子C4炸弹轰你祖宗十八代的祖坟，老子砍你全家的头，你是什么JB玩意？你这种狗杂种不仅活该楼房烂尾，你全家还要被爆杀。笑死老子了。真是你妈的个吃屎的东西。天天蹲粪坑里吃你全家拉的屎🤣。

**Reply by ColoCoffee** [reply]
> 老子乱刀劈死你全家，你全家十八代祖坟被人掘了吗？你以为你是在微博是吧？你跟你妈的个篮子样。你爹我不惯着你，你这种人出来老子第一个乱刀砍死的就是你，蠢的跟你妈的比一样，还有逼脸在这叫，你全家女性是母狗被人轮奸了。老子砍你全家的头，啥寄吧玩意。

跟你妈弱智一样刚注册github觉得自己能了？

**Reply by ColoCoffee** [reply]
> > 我要笑死了，世卫宣布新冠消失？你是多低能
> 
> 你全家女性都被老子爆杀，老子C4炸弹轰你祖宗十八代的祖坟，老子砍你全家的头，你是什么JB玩意？你这种狗杂种不仅活该楼房烂尾，你全家还要被爆杀。笑死老子了。真是你妈的个吃屎的东西。天天蹲粪坑里吃你全家拉的屎🤣。
😊笑了，你爹不会有机会买到烂尾楼了，你这辈子就好好做一头牛马吧

**Reply by ghost** [reply]
> 你这种死妈的东西就别来污染github环境了，跟你妈疯狗一样上来就咬，你祖上是妓女出生吧？让你的嘴和你妈的b一样脏？你是不是你妈被轮奸的时候生下来的让你变得这么低能？脑袋在你妈被轮奸的时候被撞坏了？来砍老子？你有这本事么？你爹马上去东京，你这fw估计连本护照都办不下来吧？

笑死你爹我了，这种蠢逼已经沦落到办本护照都光荣了吗？fw，也只有你这种fw才会把办本护照当做光荣吧？笑死你爹我了。

**Reply by ghost** [reply]
> > > 我要笑死了，世卫宣布新冠消失？你是多低能
> > 
> > 
> > 你全家女性都被老子爆杀，老子C4炸弹轰你祖宗十八代的祖坟，老子砍你全家的头，你是什么JB玩意？你这种狗杂种不仅活该楼房烂尾，你全家还要被爆杀。笑死老子了。真是你妈的个吃屎的东西。天天蹲粪坑里吃你全家拉的屎🤣。
> > 😊笑了，你爹不会有机会买到烂尾楼了，你这辈子就好好做一头牛马吧

这恐怕你爹我不能随你的愿了，你爹我早就拿到h1b工签在加州，你这种飞舞还拿本护照在哪高潮，不愧是吃你全家的屎长大的，笑死你爹我了

**Reply by ColoCoffee** [reply]
> > 你这种死妈的东西就别来污染github环境了，跟你妈疯狗一样上来就咬，你祖上是妓女出生吧？让你的嘴和你妈的b一样脏？你是不是你妈被轮奸的时候生下来的让你变得这么低能？脑袋在你妈被轮奸的时候被撞坏了？来砍老子？你有这本事么？你爹马上去东京，你这fw估计连本护照都办不下来吧？
> 
> 笑死你爹我了，这种蠢逼已经沦落到办本护照都光荣了吗？fw，也只有你这种fw才会把办本护照当做光荣吧？笑死你爹我了。

跟你这样一头畜牲讲这么多真是抱歉，毕竟你这废物出生的时候在你妈阴道里被我轮奸导致你脑袋上有几个洞，连你妈的b汉字都看不懂了

**Reply by ghost** [reply]
> > > 你这种死妈的东西就别来污染github环境了，跟你妈疯狗一样上来就咬，你祖上是妓女出生吧？让你的嘴和你妈的b一样脏？你是不是你妈被轮奸的时候生下来的让你变得这么低能？脑袋在你妈被轮奸的时候被撞坏了？来砍老子？你有这本事么？你爹马上去东京，你这fw估计连本护照都办不下来吧？
> > 
> > 
> > 笑死你爹我了，这种蠢逼已经沦落到办本护照都光荣了吗？fw，也只有你这种fw才会把办本护照当做光荣吧？笑死你爹我了。
> 
> 跟你这样一头畜牲讲这么多真是抱歉，毕竟你这废物出生的时候在你妈阴道里被我轮奸导致你脑袋上有几个洞，连你妈的b汉字都看不懂了

也只有你这种蠢逼拿了霓虹的签证才会高潮吧？有什么门槛吗？废物，都不用去面签，还拿出来在你爹面前高潮。你回去好好看看你妈吧。早就被乱刀砍死躺床上了。

**Reply by ColoCoffee** [reply]
> 



> > > > 我要笑死了，世卫宣布新冠消失？你是多低能
> > > 
> > > 
> > > 你全家女性都被老子爆杀，老子C4炸弹轰你祖宗十八代的祖坟，老子砍你全家的头，你是什么JB玩意？你这种狗杂种不仅活该楼房烂尾，你全家还要被爆杀。笑死老子了。真是你妈的个吃屎的东西。天天蹲粪坑里吃你全家拉的屎🤣。
> > > 😊笑了，你爹不会有机会买到烂尾楼了，你这辈子就好好做一头牛马吧
> 
> 这恐怕你爹我不能随你的愿了，你爹我早就拿到h1b工签在加州，你这种飞舞还拿本护照在哪高潮，不愧是吃你全家的屎长大的，笑死你爹我了

骗大家可以，别骗你自己了，刚注册的小号，第一次上github，一上来就是这种政治库，还你妈h1b工签，笑死了🫠

**Reply by ghost** [reply]
> > 
> 
> > > > > 我要笑死了，世卫宣布新冠消失？你是多低能
> > > > 
> > > > 
> > > > 你全家女性都被老子爆杀，老子C4炸弹轰你祖宗十八代的祖坟，老子砍你全家的头，你是什么JB玩意？你这种狗杂种不仅活该楼房烂尾，你全家还要被爆杀。笑死老子了。真是你妈的个吃屎的东西。天天蹲粪坑里吃你全家拉的屎🤣。
> > > > 😊笑了，你爹不会有机会买到烂尾楼了，你这辈子就好好做一头牛马吧
> > 
> > 
> > 这恐怕你爹我不能随你的愿了，你爹我早就拿到h1b工签在加州，你这种飞舞还拿本护照在哪高潮，不愧是吃你全家的屎长大的，笑死你爹我了，你是垃圾吗？拿不下h1b签证？霓虹签证？笑死我了，有什么门槛吗？fw？还东京，说出来不怕笑死你爹。
> 
> 骗大家可以，别骗你自己了，刚注册的小号，第一次上github，一上来就是这种政治库，还你妈h1b工签，笑死了🫠

笑死你爹我了，你爹我上github的时你妈都在给日本兵当慰安妇了。你爹上github来就是爆杀你全家的，回去好好看看nm，都被五马分尸扔在城门外了🤣，野狗都看不上。

**Reply by ColoCoffee** [reply]
> > > > 你这种死妈的东西就别来污染github环境了，跟你妈疯狗一样上来就咬，你祖上是妓女出生吧？让你的嘴和你妈的b一样脏？你是不是你妈被轮奸的时候生下来的让你变得这么低能？脑袋在你妈被轮奸的时候被撞坏了？来砍老子？你有这本事么？你爹马上去东京，你这fw估计连本护照都办不下来吧？
> > > 
> > > 
> > > 笑死你爹我了，这种蠢逼已经沦落到办本护照都光荣了吗？fw，也只有你这种fw才会把办本护照当做光荣吧？笑死你爹我了。
> > 
> > 
> > 跟你这样一头畜牲讲这么多真是抱歉，毕竟你这废物出生的时候在你妈阴道里被我轮奸导致你脑袋上有几个洞，连你妈的b汉字都看不懂了
> 
> 也只有你这种蠢逼拿了霓虹的签证才会高潮吧？有什么门槛吗？废物，都不用去面签，还拿出来在你爹面前高潮。你回去好好看看你妈吧。早就被乱刀砍死躺床上了。

你这畜牲知道h1b工签需要什么么？上网上多了给自己上魔怔了？出现幻觉了？是什么低能理解让你觉得你爹我会因为拿个签证而高潮啊？因为你妈被人轮奸然后砍掉四肢做成人彘的悲惨遭遇么？😄

**Reply by xiejun946** [reply]
气急败坏，还是回台湾岛舔蔡英文吧，哪天导弹飞你家祈祷不要掉下来

**Reply by ghost** [reply]
> > > > > 你这种死妈的东西就别来污染github环境了，跟你妈疯狗一样上来就咬，你祖上是妓女出生吧？让你的嘴和你妈的b一样脏？你是不是你妈被轮奸的时候生下来的让你变得这么低能？脑袋在你妈被轮奸的时候被撞坏了？来砍老子？你有这本事么？你爹马上去东京，你这fw估计连本护照都办不下来吧？
> > > > 
> > > > 
> > > > 笑死你爹我了，这种蠢逼已经沦落到办本护照都光荣了吗？fw，也只有你这种fw才会把办本护照当做光荣吧？笑死你爹我了。
> > > 
> > > 
> > > 跟你这样一头畜牲讲这么多真是抱歉，毕竟你这废物出生的时候在你妈阴道里被我轮奸导致你脑袋上有几个洞，连你妈的b汉字都看不懂了
> > 
> > 
> > 也只有你这种蠢逼拿了霓虹的签证才会高潮吧？有什么门槛吗？废物，都不用去面签，还拿出来在你爹面前高潮。你回去好好看看你妈吧。早就被乱刀砍死躺床上了。
> 
> 你这畜牲知道h1b工签需要什么么？上网上多了给自己上魔怔了？出现幻觉了？是什么低能理解让你觉得你爹我会因为拿个签证而高潮啊？因为你妈被人轮奸然后砍掉四肢做成人彘的悲惨遭遇么？😄

笑死你爹了，你是家里很垃圾吗？拿不到h1b？拿个霓虹的比签证还敢出来叫，有什么门槛吗？还东京？好牛逼哦，我好羡慕哦，纯纯土狗一条，fw，是抓紧过去吃核废水吗？fw，笑死你爹我了，也难怪，你这种杂种核废水都把你弄成了脑残，下次核爆就是核爆你全家。

**Reply by ColoCoffee** [reply]
> 老子乱刀劈死你全家，你全家十八代祖坟被人掘了吗？你以为你是在微博是吧？你跟你妈的个篮子样。你爹我不惯着你，你这种人出来老子第一个乱刀砍死的就是你，蠢的跟你妈的比一样，还有逼脸在这叫，你全家女性是母狗被人轮奸了。老子砍你全家的头，啥寄吧玩意。

果然是意淫自己在美国享受快乐生活的低能犬，连骂人都这么像低能

**Reply by ghost** [reply]
> > 老子乱刀劈死你全家，你全家十八代祖坟被人掘了吗？你以为你是在微博是吧？你跟你妈的个篮子样。你爹我不惯着你，你这种人出来老子第一个乱刀砍死的就是你，蠢的跟你妈的比一样，还有逼脸在这叫，你全家女性是母狗被人轮奸了。老子砍你全家的头，啥寄吧玩意。
> 
> 果然是意淫自己在美国享受快乐生活的低能犬，连骂人都这么像低能

你这种飞舞也只能用这种方式来安慰自己，还日本签，好骄傲哦，有什么门槛吗？fw？你祖宗十八代都光宗耀祖了，你全家女性都被派去给日本兵当慰安妇了。

**Reply by ColoCoffee** [reply]
> 

我真的要被你这低能笑死，h1b和家庭情况有你妈的半点关系？别做低能了好么？祖宗十八代当狗的东西。2022年8月15号第一天加入github，拿h1b？😂😂😂😂

**Reply by ColoCoffee** [reply]
> > > 老子乱刀劈死你全家，你全家十八代祖坟被人掘了吗？你以为你是在微博是吧？你跟你妈的个篮子样。你爹我不惯着你，你这种人出来老子第一个乱刀砍死的就是你，蠢的跟你妈的比一样，还有逼脸在这叫，你全家女性是母狗被人轮奸了。老子砍你全家的头，啥寄吧玩意。
> > 
> > 
> > 果然是意淫自己在美国享受快乐生活的低能犬，连骂人都这么像低能
> 
> 你这种飞舞也只能用这种方式来安慰自己，还日本签，好骄傲哦，有什么门槛吗？fw？你祖宗十八代都光宗耀祖了，你全家女性都被派去给日本兵当慰安妇了。

确实只有你种底层低能废狗会觉得别人拿个签字就觉得自己牛逼了

**Reply by ColoCoffee** [reply]
> > > 老子乱刀劈死你全家，你全家十八代祖坟被人掘了吗？你以为你是在微博是吧？你跟你妈的个篮子样。你爹我不惯着你，你这种人出来老子第一个乱刀砍死的就是你，蠢的跟你妈的比一样，还有逼脸在这叫，你全家女性是母狗被人轮奸了。老子砍你全家的头，啥寄吧玩意。
> > 
> > 
> > 果然是意淫自己在美国享受快乐生活的低能犬，连骂人都这么像低能
> 
> 你这种飞舞也只能用这种方式来安慰自己，还日本签，好骄傲哦，有什么门槛吗？fw？你祖宗十八代都光宗耀祖了，你全家女性都被派去给日本兵当慰安妇了。

你这种低能下次在网上查东西拿来意淫的时候，麻烦查全了😊

**Reply by ghost** [reply]
> > > > 老子乱刀劈死你全家，你全家十八代祖坟被人掘了吗？你以为你是在微博是吧？你跟你妈的个篮子样。你爹我不惯着你，你这种人出来老子第一个乱刀砍死的就是你，蠢的跟你妈的比一样，还有逼脸在这叫，你全家女性是母狗被人轮奸了。老子砍你全家的头，啥寄吧玩意。
> > > 
> > > 
> > > 果然是意淫自己在美国享受快乐生活的低能犬，连骂人都这么像低能
> > 
> > 
> > 你这种飞舞也只能用这种方式来安慰自己，还日本签，好骄傲哦，有什么门槛吗？fw？你祖宗十八代都光宗耀祖了，你全家女性都被派去给日本兵当慰安妇了。
> 
> 确实只有你种底层低能废狗会觉得别人拿个签字就觉得自己牛逼了

你爹会觉得你这种废狗牛逼？你全家跟你妈比一样，拿个日本签还敢拿出来叫？有什么门槛吗？fw。你这种篮子是拿不到美签只能用这种方式来安慰自己吗？也只有你这种纯底层脑瘫fw狗全家死绝的玩意的底层fw才会当做光荣。对了，日本那保证金不会是你全家女性当慰安妇卖比的钱吧,233333,这种家庭也真是可怜。

**Reply by ghost** [reply]
> 你这种死妈的东西就别来污染github环境了，跟你妈疯狗一样上来就咬，你祖上是妓女出生吧？让你的嘴和你妈的b一样脏？你是不是你妈被轮奸的时候生下来的让你变得这么低能？脑袋在你妈被轮奸的时候被撞坏了？来砍老子？你有这本事么？你爹马上去东京，你这fw估计连本护照都办不下来吧？

办本护照有什么门槛吗？让你这野狗高潮成这样？？？还是这种死全家十八代的没办过护照？拿个日本签去东京，笑死你爹我了，有什么门槛吗？也只有你这种底层烂逼，全家死绝的fw才会去日本讨一口饭吃？什么底层烂逼会去日本？笑死你爹了。

**Reply by ghost** [reply]
> 你这种死妈的东西就别来污染github环境了，跟你妈疯狗一样上来就咬，你祖上是妓女出生吧？让你的嘴和你妈的b一样脏？你是不是你妈被轮奸的时候生下来的让你变得这么低能？脑袋在你妈被轮奸的时候被撞坏了？来砍老子？你有这本事么？你爹马上去东京，你这fw估计连本护照都办不下来吧？

估计你这种全家死绝女的当慰安妇男的被乱刀砍死的家庭，去了哪个会社当人肉电池？还是去了哪所野鸡大学？233333，东京，说的你爹我好羡慕啊，你妈卖比给日本兵当慰安妇的钱够买机票吗？2333333.

**Reply by ghost** [reply]
> 你这种死妈的东西就别来污染github环境了，跟你妈疯狗一样上来就咬，你祖上是妓女出生吧？让你的嘴和你妈的b一样脏？你是不是你妈被轮奸的时候生下来的让你变得这么低能？脑袋在你妈被轮奸的时候被撞坏了？来砍老子？你有这本事么？你爹马上去东京，你这fw估计连本护照都办不下来吧？

估计也只有你这种你妈当慰安妇和日本兵杂交生出来的蠢逼杂种才会去霓虹要饭，你妈得和多少个日本兵交配才能赚够你那机票钱？日本东京我好羡慕哦？也难怪，你妈当天当完慰安妇就被日本兵砍了头，所以你现在去霓虹找你的日本爹吗？23333333，日本会社工资好高哦，你爹我好羡慕哦，笑死你爹了，你妈你全家女性以及你十八代祖宗都处在被日本兵爆杀的无限轮回之中，来回砍你全家的头。
